### DISCUSSION TITLE
Signature request on Coinbase Smart Wallet

### DISCUSSION THREAD

**Discussion by Ernest-Tribe3** [wevm/wagmi#4108] [Q&A]
`const signature = await walletClient?.signMessage({
      message: message,
    })`

When im signing message using Coinbase's smart wallet, the signature's format is significantly different with normal eth wallets. Is there any ways parsing or reformatting the signature? Thanks!

**Comment by bazzilic** [comment]
We are encountering the same issue, haven't found a solution yet.

**Comment by cheziyi** [comment]
So that is a SIWE Message, and should be verified as such: https://viem.sh/docs/siwe/actions/verifySiweMessage
